.rubocop.yml

```diff
@@ -58,17 +74,16 @@ Style/IfInsideElse:
 Style/RedundantReturn:
   Enabled: false
 
-Metrics/AbcSize:
+# As of July 2016, there is enough disagreement in our codebase, on our team,
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
+Style/SpaceAroundEqualsInParameterDefault:
   Enabled: false
 
-Metrics/LineLength:
+# As of July 2016, there is enough disagreement in our codebase, on our team,
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
+Style/StringLiterals:
```

Comment:
Honestly, I'd like to see the rule enabled at some point, there is definite benefit to knowing the string is a vanilla string versus a complex string at a glance. Especially if we could reenable the rule without messing up git blame.